sources/models/shared-defaults/oci-defaults-docker-cri.toml

```diff
@@ -0,0 +1,10 @@
+[metadata.settings.oci-defaults]
```

Comment:
This file is named `oci-defaults-docker-cri`, but `cri` only applies to Kubernetes. Please drop that part of the name.